frontend/src/scenes/data-warehouse/editor/multitabEditorLogic.tsx

```diff
@@ -1087,6 +1087,8 @@ export const multitabEditorLogic = kea<multitabEditorLogicType>([
                 if (searchParams.open_view) {
                     // Open view
                     const viewId = searchParams.open_view
+
+                    await dataWarehouseViewsLogic.asyncActions.loadDataWarehouseSavedQueries()
```

Comment:
Lets do a check to make sure these aren't already loaded in

```python
if len(values.dataWarehouseSavedQueries) == 0:
    await dataWarehouseViewsLogic.asyncActions.loadDataWarehouseSavedQueries()
```

Reply:
I thought about this, but I was wondering if they actually didn't have any or deleted them and this looping somehow.  I'll give it a try.

Reply:
Ok, looks ok, we wipe out the query params once they are consumed.